CONTRIBUTING.md

```diff
@@ -79,15 +80,15 @@ which clearly explains the setup and configuration of NVM.
 
 ### Cloning vs forking
 
-To be able to [clone](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) this repository and contribute you will need to be given write access to the repository. This is reserved for New Relic Doc Writers. Contact the Developer Enablement team (#help-deven-websites Slack channel) if you need write access.
+To be able to [clone](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) this repository and contribute you will need to be given write access to the repository. This is reserved for New Relic Doc Writers. Contact the Docs Engineering team (#help-deven-websites Slack channel) if you need write access.
```

Comment:
Slightly blocked until slack channel is up and running. Just need to change the slack channel name once that is set up